config/examples/rolling_updates.yml

```diff
@@ -14,22 +13,17 @@ base: &default
   policy_registry: true
   proxy_private_base_path: true
   service_mesh_integration: false
-  api_as_product: false
+  apicast_v2: true
+  published_service_plan_signup: true
+  apicast_oidc: true
+  provider_sso:
+  billable_contracts:
```

Comment:
Where did you get this list from? Were all these features implemented but not enabled? I don't see them in the production config in the deploy repo

Reply:
I was looking at the wrong file  🫠